wp-content/themes/core/assets/css/src/admin/acf/acf-image.pcss

```diff
@@ -0,0 +1,9 @@
+/* hide the 'No Image Selected' text for the image field */
+.block-editor .edit-post-sidebar .acf-image-uploader p {
```

Comment:
A few asks for the selector blocks (this file and below):
1. Is the selector specificity required or can it be reduced at all?
2. Please nest the selectors to reduce the repetition and make the styles easier to grep. Example:
```
.block-editor {

	.edit-post-sidebar {

		/* hide the 'No Image Selected' text for the image field */
		.acf-image-uploader p {
			font-size: 0px
		}

		/* Give the image field a border */
		.acf-field-image .acf-input {
			border: 1px solid #ccd0d4; /* WP Core border color */
			padding: 12px;
		}
	}
}
```